src/main/java/pivotaltracker/ui/pages/DashboardPage.java

```diff
@@ -26,6 +26,15 @@
     @FindBy(css = "[aria-label='Profile Dropdown']")
     private WebElement profileBtn;
 
+    @FindBy(css = "[class='Dashboard__Tabs__tab']")
+    public WebElement projectTab;
```

Comment:
Do these web elements need to be public?

Reply:
solve